docs/swagger/clob-routes.yml

```diff
@@ -0,0 +1,275 @@
+      description: 'Create one or more orders.'
+      operationId: 'postOrders'
+      consumes:
+        - 'application/json'
+      produces:
+        - 'application/json'
+      parameters:
+        - in: 'body'
+          name: 'body'
+          description: 'Request body.'
+          required: true
+          schema:
+            $ref: '#/definitions/ClobPostOrderRequest'
+      responses:
+        '200':
+          description: 'Successful response.'
+          schema:
+            $ref: '#/definitions/ClobPostOrderResponse'
+        '400':
+          description: 'Bad request response.'
+    get:
+      tags:
+        - 'clob'
+      summary: 'Get One Or More Orders'
+      description: 'Get the information of one, several or all orders.'
+      operationId: 'orders'
+      consumes:
+        - 'application/json'
+      produces:
+        - 'application/json'
+      parameters:
+        - in: 'body'
+          name: 'body'
+          description: 'Request body.'
+          required: true
+          schema:
+            $ref: '#/definitions/ClobGetOrderRequest'
+      responses:
+        '200':
+          description: 'Successful response.'
+          schema:
+            $ref: '#/definitions/ClobGetOrderResponse'
+        '400':
+          description: 'Bad request response.'
+        '404':
+          description: 'Not found response.'
+    delete:
+      tags:
+        - 'clob'
+      summary: 'Cancel One Or More Orders Open Orders'
+      description: 'Cancel one, several or all open orders.'
+      operationId: 'deleteOrders'
+      consumes:
+        - 'application/json'
+      produces:
+        - 'application/json'
+      parameters:
+        - in: 'body'
+          name: 'body'
+          description: 'Request body.'
+          required: true
+          schema:
+            $ref: '#/definitions/ClobDeleteOrderRequest'
+      responses:
+        '200':
+          description: 'Successful response.'
+          schema:
+            $ref: '#/definitions/ClobDeleteOrderResponse'
+        '400':
+          description: 'Bad request response.'
+        '404':
+          description: 'Not found response.'
+  /clob/orders/open:
+    get:
+      tags:
+        - 'clob'
+      summary: 'Get One Or More Open Orders'
+      description: 'Get the information of one, several or all open orders.'
+      operationId: 'clob.getOpenOrders'
+      consumes:
+        - 'application/json'
+      produces:
+        - 'application/json'
+      parameters:
+        - in: 'body'
+          name: 'body'
+          description: 'Request body.'
+          required: true
+          schema:
+            $ref: '#/definitions/SerumGetOpenOrdersRequest'
+      responses:
+        '200':
+          description: 'Successful response.'
+          schema:
+            $ref: '#/definitions/SerumGetOpenOrdersResponse'
+        '400':
+          description: 'Bad request response.'
+        '404':
+          description: 'Not found response.'
+  /clob/orders/filled:
```

Comment:
the following routes don't exist:
- GET orders/open
- GET orders/filled
- POST settleFunds

Reply:
Right.
They were created with the hope that they'll be used by Injective or future CLOB conector.
But I think it makes sense to remove them now. Connectors that require them in the future can add them.